transport.go

```diff
@@ -27,12 +27,12 @@ func (t HttpTransport) Exec(conn *Conn, q Query, readOnly bool) (res string, err
 	client := &http.Client{Timeout: t.Timeout}
 	if readOnly {
 		if len(query) > 0 {
-			query = "?query=" + query
+			query = getDb(conn) + "query=" + query
 		}
 		resp, err = client.Get(conn.Host + query)
```

Comment:
`host + query` should be extracted to func (`getQueryBaseURL` or something) and covered with test